<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="mainPlot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>261</y>
     <width>751</width>
     <height>311</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>741</width>
     <height>311</height>
    </size>
   </property>
  </widget>
  <widget class="QGroupBox" name="consoleBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>751</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>751</width>
     <height>0</height>
    </size>
   </property>
   <property name="title">
    <string>Console</string>
   </property>
   <widget class="QTextBrowser" name="console">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>741</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>751</width>
     <height>71</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>741</width>
     <height>71</height>
    </size>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QWidget" name="outputWidget" native="true">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>0</height>
     </size>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>441</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="outDirLayout">
      <item>
       <widget class="QLabel" name="outDirLabel">
        <property name="text">
         <string>Output Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outDirInput"/>
      </item>
      <item>
       <widget class="QToolButton" name="browseButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="realTimeBox">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>271</width>
      <height>0</height>
     </size>
    </property>
    <property name="title">
     <string>Real Time</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>51</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="realTimeLayout2">
      <item>
       <widget class="QLabel" name="realHLabel">
        <property name="text">
         <string>h (m)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="realHDisplay">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>53</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="realTimeLayout1">
      <item>
       <widget class="QLabel" name="realULabel">
        <property name="text">
         <string>u (m/s)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="realUDisplay">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="realTimeLayout3">
      <item>
       <widget class="QLabel" name="timeElapsedLabel">
        <property name="text">
         <string>Time (h)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="timeElapsedDisplay">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>65</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="realTimeLayout3_2">
      <item>
       <widget class="QLabel" name="tempLabel">
        <property name="text">
         <string>Temp (C)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="tempDisplay">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>171</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QGroupBox" name="sustainedSettings">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>171</height>
     </size>
    </property>
    <property name="title">
     <string>Sustained Run</string>
    </property>
    <widget class="QPushButton" name="sustainedStartButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>201</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="targetULayout">
      <item>
       <widget class="QLabel" name="targetULabel">
        <property name="text">
         <string>Target u_{rms}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetUUnits">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="actualULayout">
      <item>
       <widget class="QLabel" name="actualULabel">
        <property name="text">
         <string>Actual u_{rms}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="actualUDisplay">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="actualUUnits">
        <property name="text">
         <string>m/s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>201</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="targetULayout_3">
      <item>
       <widget class="QLabel" name="targetTimeLabel">
        <property name="text">
         <string>Exp. runtime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="targetTimeEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetTimeUnits">
        <property name="text">
         <string>h</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="periodSettings">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>161</height>
     </size>
    </property>
    <property name="title">
     <string>Period Determination</string>
    </property>
    <widget class="QPushButton" name="periodStartButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>201</width>
       <height>35</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="targetTLayout">
      <item>
       <widget class="QLabel" name="targetTLabel">
        <property name="text">
         <string>Target T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="targetTEdit">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="targetTUnits">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="actualTLayout">
      <item>
       <widget class="QLabel" name="actualTLabel">
        <property name="text">
         <string>Actual T</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="actualTDisplay">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="actualTUnits">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="paramTabs">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>331</width>
      <height>161</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="physicalTab">
     <attribute name="title">
      <string>Physical</string>
     </attribute>
     <widget class="QLabel" name="densityLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Water density (kg/m^3)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="densityEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="areaFactorLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Res-to-Duct factor</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="areaFactorEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1.7778</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="numericalTab">
     <attribute name="title">
      <string>Numerical</string>
     </attribute>
     <widget class="QLabel" name="fbackVelLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Feedback velocity</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fbackVelEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="fbackAccLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Feedback acceleration</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fbackAccEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="pressureSampleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pressure sample rate (hz)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="pressureSampleEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="sinSampleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sin-wave sample rate (hz)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sinSampleEdit">
      <property name="geometry">
       <rect>
        <x>222</x>
        <y>70</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rotorTab">
     <attribute name="title">
      <string>Rotors</string>
     </attribute>
     <widget class="QLabel" name="initBurstLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Initial burst power (%)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="initBurstEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="initLengthEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="initLengthLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Initial burst length (s)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="initPowerEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>50</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="initPowerLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Initial power (%)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="initChannelsEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0,1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="initChannelsLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Channels (#,#,#,...,#)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <zorder>splitter_3</zorder>
  <zorder>console</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
